wiki/History_of_osu!/1/en.md

```diff
@@ -145,8 +165,9 @@ Progress was made on **OpenGL implementation** and the **upcoming game mode debu
 November
 --------
 
-<img src="Fruit ryuuta.png" title="fig:The catcher of Catch the Beat" alt="The catcher of Catch the Beat" width="100" /> November saw the introduction of a brand new play mode, **"[Catch the Beat](Catch_the_Beat "wikilink")". OpenGL support debuted**, thus allowing osu! to run on most systems. Other new features in this update packed month included **in-game friend list support, private chat tabs, a new tutorial/[offset wizard](Options#Offset_Wizard "wikilink") and osu!direct**; an in-game beatmap downloader for supporters.
+![The catcher of Catch the Beat.](img/Fruit_ryuuta.png "The catcher of Catch the beat.")
 
+ November saw the introduction of a brand new play mode, **"[Catch the Beat](Catch_the_Beat "wikilink")". OpenGL support debuted**, thus allowing osu! to run on most systems. Other new features in this update packed month included **in-game friend list support, private chat tabs, a new tutorial/ [offset wizard](Options#Offset_Wizard "wikilink") and osu!direct**; an in-game beatmap downloader for supporters.
```

Comment:
if anything, please don't change this to say "osu!catch", keep it as-is.

replace line with these fixes
```
November saw the introduction of a brand new play mode, **"[Catch the Beat](/wiki/Game_Modes/osu!catch")". OpenGL support debuted**, thus allowing osu! to run on most systems. Other new features in this update packed month included **in-game friend list support, private chat tabs, a new tutorial/ [offset wizard](/wiki/Options#offset-wizard) and osu!direct**; an in-game beatmap downloader for supporters.
```